man/systemctl.xml

```diff
@@ -876,6 +876,42 @@ kobject-uevent 1 systemd-udevd-kernel.socket systemd-udevd.service
             a similar filter for messages and might be more
             convenient.
             </para>
+
+
+            <example>
+              <title>Example output from systemctl status </title>
+
+              <programlisting># systemctl status bluetooth
+$ systemctl status bluetooth
+● bluetooth.service - Bluetooth service
+   Loaded: loaded (/lib/systemd/system/bluetooth.service; enabled; vendor preset: enabled)
+   Active: active (running) since Wed 2017-01-04 13:54:04 EST; 1 weeks 0 days ago
+     Docs: man:bluetoothd(8)
+ Main PID: 930 (bluetoothd)
+   Status: "Running"
+    Tasks: 1
+   Memory: 648.0K
+      CPU: 435ms
+   CGroup: /system.slice/bluetooth.service
+           └─930 /usr/lib/bluetooth/bluetoothd
+
+# Recent logs from the bluetooth service would appear here</programlisting>
+
+            <para>The Loaded: line in the output will show <literal>loaded</literal> if the unit has been loaded into
+              memory and <literal>not-found</literal> if service can't be found.  Along with showing the path to the
+              unit file, this line will also show the enablement state.  Enabled commands start at boot.  See the full
+              table of possible enablement states in documentation for the <command>is-enabled</command> command.
+            </para>
+
+            <para>The Active: line shows active state.  The value is usually <literal>active</literal> or
+              <literal>inactive</literal>. Active could mean started, bound, plugged in, etc depending on the unit
+              type.  The unit could also be in process of changing states, reporting a state of
+              <literal>activating</literal> or <literal>deactivating</literal>.  A special <literal>failed</literal>
+              state is entered when the service failed in some way, such as a crash, exiting with an error code or
+              timing out.  If the failed state is entered the cause will be logged for later reference.  </para>
+
+            </example>
+
```

Comment:
The text is nice. It describes the common case and refers to the right place for the full list.

I think it would be nice to also mention the color of the dot.